content/en/cd-as-a-service/tasks/tenants/switch-tenants.md

```diff
@@ -0,0 +1,38 @@
+---
+title: View and Switch Tenants
+description: >
+  View and switch tenants in your Armory CD-as-a-Service organization.
+---
+
+## {{% heading "prereq" %}}
+
+* You have a [custom Tenant Admin role]({{< ref "cd-as-a-service/tasks/iam/create-role#user-role-examples" >}}) that applies to more than one tenant **or** you have the Organization Admin role.
+* You have installed the CLI and are logged into CD-as-a-Service.
+
+## View tenants
+
+You can use the CLI to view your tenants:
+
+{{< prism lang="bash" line-numbers="true" >}}
+armory config get
+{{< /prism >}}
+
+Output shows the list of tenants. When you only have the default tenant, the output is:
+
+{{< prism lang="bash" line-numbers="true" >}}
+tenants:
+ - main
+{{< /prism >}}
+
+You can also use the UI to view the list of tenants you can access. See the [Switch tenants](#switch-tenants) section.
+
+## Switch tenants
```

Comment:
I'd suggest adding - 
Non-Admin Users will only see tenants that they belong to. If they don't have a role belonging to a particular tenant, then they would not see those tenants.